lib/dynflow/persistence_adapters/sequel_migrations/018_add_uuid_column.rb

```diff
@@ -41,21 +42,21 @@ def with_foreign_key_recreation(&block)
   up do
     if database_type.to_s.include?('postgres')
       with_foreign_key_recreation do
```

Comment:
It seems, this should be 
```suggestion
      helper.with_foreign_key_recreation do
```

I guess there are more places now where `helper` is needed now.